docs/ConfigureScalarDL.md

```diff
@@ -2,32 +2,39 @@
-This optional method is highly recommended for handling credentials in the production environment.
+### Kubernetes Secrets and Schema Loading reference
 
-You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
+You can refer to the table below to get values to set create Kubernetes Secrets and update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file properties file according to the database you chose for deployment.
 
-```
-# Cosmos DB
-    # username=<Account name>
-    # password=<Cosmos DB account primary master key>
+| Database  | database | contactPoints              | username          | password                             | dynamoBaseResourceUnit | cosmosBaseResourceUnit | 
+|:----------|----------|----------------------------|-------------------|--------------------------------------|------------------------|------------------------|
+| DynamoDB  | dynamo   | REGION                     | AWS_ACCESS_KEY_ID | AWS_ACCESS_SECRET_KEY                | 10                     | N/A                    |
+| Cosmos DB | cosmos   | Cosmos DB account endpoint | N/A               | Cosmos DB account primary master key | N/A                    | 400                    |
+| JDBC      | jdbc     | JDBC_CONNECTION_URL        | USERNAME          | PASSWORD                             | N/A                    | N/A                    |
 
-# DynamoDB
-    # username=<AWS_ACCESS_KEY_ID>
-    # password=<AWS_ACCESS_SECRET_KEY>
 
+### Create Kubernetes Secrets
+
+Kubernetes Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key.
+This optional method is highly recommended for handling credentials in the production environment.
+You can refer to the [section](#kubernetes-secret-and-schema-loading-reference) to get the values for `username` and `password` for the database you chose.
+You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
+
+
+```
 kubectl create secret generic scalardl --from-literal db-username=<username> --from-literal db-password=<password>
 ```
+Note-:
 
-## Update Configuration Files 
+For Cosmos DB, use the account name as `username` since we can't leave the `db-uername` property empty.
 
-### Cosmos DB
-
-#### Configure schema-loading-custom-values
+### Configure schema-loading-custom-values
 
 To create a Scalar DL schema in Cosmos DB, you need to update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file.
+You can refer to the [section](#kubernetes-secret-and-schema-loading-reference) to check which values are needed based on the database used.
+If any configuration property value is defined as `N/A` in the reference table for your database, you can remove that property from the configuration file.
```

Comment:
```suggestion
If any configuration property value is defined as `N/A` in the reference table for your database, you need to remove that property from the configuration file.
```

Reply:
I will update it as per suggestion. Thank you